tomviz/GradientOpacityWidget.cxx

```diff
@@ -66,6 +66,13 @@ GradientOpacityWidget::GradientOpacityWidget(QWidget* parent_)
   hLayout->setContentsMargins(0, 0, 35, 0);
 
   setLayout(hLayout);
+
+  // Delay setting of the device pixel ratio until after the QVTKOpenGLWidget
+  // widget has been set up.
+  QTimer::singleShot(0, [=] {
+    int dpi = m_qvtk->physicalDpiX() * m_qvtk->devicePixelRatioF();
```

Comment:
Why are you using the float version of device pixel ratio, and then turning it back into an int? This can only be 1 or 2, so I generally use the int version. Everything else makes sense to me.